lib/manageiq/environment.rb

```diff
@@ -0,0 +1,104 @@
+require 'fileutils'
+require 'pathname'
+
+module ManageIQ
+  module Environment
+    APP_ROOT = Pathname.new(__dir__).join("../..")
+
+    def self.ensure_config_files
+      config_files = {
+        "certs/v2_key.dev"        => "certs/v2_key",
+        "config/cable.yml.sample" => "config/cable.yml",
+        "config/database.pg.yml"  => "config/database.yml",
```

Comment:
I like how the hashrockets demonstrate the copying from source to destination better than 1.9 syntax would do here.  :trollface: @chrisarcand